mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -53,16 +53,6 @@ cfg_wasm32! {
 fn test_rpc() {
     let (_, mm, _dump_log, _dump_dashboard) = mm_spat();
 
-    let no_method = block_on(mm.rpc(&json! ({
-        "userpass": mm.userpass,
-        "coin": "RICK",
-        "ipaddr": "electrum1.cipig.net",
-        "port": 10017
-    })))
-    .unwrap();
-    assert!(no_method.0.is_server_error());
-    assert_eq!((no_method.2)[ACCESS_CONTROL_ALLOW_ORIGIN], "http://localhost:4000");
```

Comment:
This was testing unknown methods on the legacy dispatcher (not very useful to cover to be honest). The behavior for unknown RPCs in the legacy dispatcher has changed: if a method isn't found there, we now try to find it in the current dispatcher.